astropy/wcs/wcsapi/utils.py

```diff
@@ -48,42 +53,54 @@ def wcs_info_str(wcs):
     pixel_nam_width = max(9, max(len(x) for x in wcs.pixel_axis_names))
     pixel_siz_width = max(9, len(str(max(array_shape))))
 
+    # fmt: off
     s += (('{0:' + str(pixel_dim_width) + 's}').format('Pixel Dim') + '  ' +
             ('{0:' + str(pixel_nam_width) + 's}').format('Axis Name') + '  ' +
             ('{0:' + str(pixel_siz_width) + 's}').format('Data size') + '  ' +
             'Bounds\n')
+    # fmt: on
```

Comment:
There are several portions of this file like this, where `black` made these way worse to read. So I reverted the `black` changes and turned it off.

I think all these sections could probably be refactored, but I don't know enough about them to attempt to do so.